samples/MvcApp/Global.asax.cs

```diff
@@ -16,17 +16,15 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
 
-            Application.AddSystemWebAdapters()
+            this.AddSystemWebAdapters()
```

Comment:
I see why you want to do this, but an extension method off of `this` feels weird... we'll want to discuss that at the API review

Reply:
Yea, I agree. The alternative would be to just pass `this` to the method but then maybe we need to re-name the class so that it's friendlier to access the methods as statics.

Reply:
What do you think of `SystemWebAdapterConfiguration.AddSystemWebAdapters(this)`?